compact/mrich.xml

```diff
@@ -4,17 +4,18 @@
     <constant name="MRICH_rmin"   value="10*cm"/>
     <constant name="MRICH_rmax"   value="BackwardPIDRegion_rmax"/>
     <constant name="MRICH_length" value="BackwardPIDRegion_length"/>
-    <constant name="MRICH_zmin"   value="BackwardPIDRegion_zmin"/>
+    <constant name="MRICH_zmax"   value="BackwardPIDRegion_zmax"/>
+    <constant name="MRICH_zmin"   value="MRICH_zmax-MRICH_length"/>
```

Comment:
@c-dilks This is how I'm pushing the MRICH up against zmax. Wasn't as cumbersome as I had feared (and definitely avoided having to mess with jinja).

Reply:
Makes sense, I will update #158 to be consistent. Btw, I get `172.470` for `BackwardPIDRegion_zmax` here, but the menagerie says `172.7`. It's not a big enough difference to matter, at least for the pfRICH, so I'll leave it up to you if you want to update it here.

Reply:
Just need to change `CentralTrackingRegionN_zmax`

Reply:
I'm deferring to the `BackwardPIDRegion_zmax` as provided by the parametrization, so I'm not going to hard code it here.